nailgun/entities.py

```diff
@@ -5073,7 +5092,6 @@ def __init__(self, server_config=None, **kwargs):
                 unique=True
             ),
             'ptable': entity_fields.OneToManyField(PartitionTable),
-            'config_template': entity_fields.OneToManyField(ConfigTemplate),
```

Comment:
Do we care?
That and line 5342 are also removed in #724 (which was submitted after this one, I know). I would probably focus only on new/changed functionality here, and let other PR deal with removal of config_templates.
(And also, I never know if I can trust github to do the right thing in cases like that)

Reply:
cool, I can close this in favor of PR#724 then